vuepress/docs/next/tutorials/ecr/deploy-components-without-bundle.md

```diff
@@ -55,58 +63,47 @@ If for example during setup wizard you chose a custom organization `myorg` and t
  </plugin>
 ```
 
-> **Note**
->
-> Output image name can also be set by customizing the `./mvnw` command using the `-Djib.to.image`
-> parameter. For example, if you want to build an image with organization `myneworg`, name `myapp` and version `latest` you can do
-```
- ./mvnw -Pprod clean package jib:dockerBuild -Djib.to.image=myneworg/myapp:latest
-```
 
-> **Warning**
->
-> If you change the target image of the docker build, remember to update
-> the plugin metadata in the bundle accordingly.
 
-## 2. Publish the Docker image to Docker registry (DockerHub or equivalent)
+::: tip Warning
+If you change the target image of the docker build, remember to update
+the plugin metadata in the bundle accordingly. You will also have to change the plugin custom resource when you deploy the plugin below in the `bundle/plugins` folder to point to the correct image.
```

Comment:
"You will also have to change the plugin custom resource to point to the correct image when you deploy the plugin in the `bundle\plugins` folder" ?
lock down custom resource capitalization rule(s)

Reply:
reworded and capitalized.